lalrpop/Cargo.toml

```diff
@@ -33,26 +33,21 @@ term = { version = "0.7", default_features = false }
 tiny-keccak = { version = "2.0.2", features = ["sha3"] }
 unicode-xid = { version = "0.2", default_features = false }
 
-# This dependency is only needed for binary builds, if you use LALRPOP as
-# library, disable it in your project by setting default_features = false.
 pico-args = { version = "0.5", default_features = false, optional = true }
 
+lalrpop-util = { path = "../lalrpop-util", version = "0.19.10" }
 
 [dev-dependencies]
 rand = "0.8"
 
-[dependencies.lalrpop-util]
-path = "../lalrpop-util"
-version = "0.19.10" # LALRPOP
-
 [features]
-default=["lexer", "pico-args"]
```

Comment:
Pretty sure this is a non-issue, since this PR is targeting 0.20, but just to highlight anyways, this is a semver breaking change, so it should definitely only go in a breaking update, and should probably get highlighted in a changelog.

I agree with the logic on removing the default feature, for the reasons @ithinuel expressed elsewhere in this PR, I just want to make sure that we highlight and communicate it properly on the off-chance that someone is affected.  One group that this might hit is anyone doing `cargo install lalrpop` in an automated script somewhere.  Presumably when they update from 0.19 to 0.20 they'll retest their script and update the command line, but it never hurts to be cautious and be able to say "look at the clear changelog" if anyone is impacted by the change to defaults.

Reply:
Since semver regards 0.x minor version changes as breaking change, I don't worry about it.
The concern is the standard user group in rust community don't expect `cargo install` has complicated requirements with more arguments.
It might be acceptable because lalrpop is more like professional dev tool rather than end user application. I personally will avoid this kind of user experience for any end-user applications.

Reply:
Yes, I think it's fine to put this in 0.20, since it's a semver change from 0.19.  My only point is that it's something worth highlighting in the 0.20 release notes.